docs/developer-guide/etendo-copilot/installation.md

```diff
@@ -47,10 +50,11 @@ The getting started guide for the Copilot API is a tool that allows interaction
     | **Environment Variable**   | **Options**  | **Info** |
     | -------------------------- | -------------| -------- |
     | COPILOT_PORT           | `5000`   | **Required** The copilot port can be defined by the user |
-    | OPENAI_API_KEY         | `***********************` | **Required** You can get it from [OpenAI API keys](https://platform.openai.com/account/api-keys){target="_blank"} |
+    | OPENAI_API_KEY         | `***********************` | **Required** You can get it from [OpenAI API keys](https://platform.openai.com/account/api-keys){target="_blank"} if you use an own API key or get in touch with Etendo support team |
```

Comment:
You can use an API Key of your own, or you can contact the Etendo support team to obtain one. ?